Regression/WarpX-tests.ini

```diff
@@ -1410,7 +1442,7 @@ inputFile = Examples/Modules/laser_injection_from_file/analysis.py
 aux1File = Examples/Modules/laser_injection_from_file/inputs.2d_test_txye
 customRunCmd = ./analysis.py
 dim = 2
-addToCompileString =
+addToCompileString = USE_OPENPMD=FALSE
```

Comment:
Hi Axel thanks for this nice PR!
Can you add some plots for future reference of what comes out of this example in this PR's comments?
PS: sorry I got a bit confused ... is openPMD default in the tests?

Reply:
Thanks Diana!
Thanks for the reminder, yes definitely planned to add some plots with adequate resolution.

openPMD was activated by default already in CI for all but the Cartesian builds. The reason is rooted in https://github.com/openPMD/openPMD-api/issues/694 which means that as of today, I cannot use an MPI-enabled openPMD-api in a serial context (because `mpi.h` gets included unconditionally then). In order to avoid building and managing two variants of the library, I for now just disable openPMD in the CI builds with noMPI.
Nevertheless, using a serial openPMD-api library with a serial WarpX already works. Just need to abstract the issue mentioned above at some point to be even more flexible.